### PR TITLE
KATI-51_PatientCondition/SymptomDetailsUI

### PR DESCRIPTION
**Pull request by toannguyen71195**

### CODE REVIEW

File: app/src/main/java/com/kms/cura/view/activity/PatientHealthDetailActivity.java [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+        }
+    }
+
+
+    @Override
+    public boolean onMenuItemClick(MenuItem item) {
+        if (item.getItemId() == R.id.btnEdit) {
+            changeMode();
+        }
+        if (item.getItemId() == R.id.btnDone) {
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/activity/PatientHealthDetailActivity.java]: if here and above should be change to switch case

File: app/src/main/java/com/kms/cura/view/activity/PatientHealthDetailActivity.java [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+            // make change in database and refresh CurrentUser
+            healthEntity.setComment(edtComment.getText().toString());
+            healthEntity.setStartDate(toDateSQL(startYear, startMonth, startDay));
+            if (endDateSet) {
+                healthEntity.setEndDate(toDateSQL(endYear, endMonth, endDay));
+            }
+            setDataOnView(healthEntity);
+            changeMode();
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/activity/PatientHealthDetailActivity.java]: Also this block of code should be in a method name something like doneButtonPressed

**Comment by hieumai** [comment]
OK from me. @huuduy216 and @vtnlinh95  please start verifying this

**Comment by vtnlinh95** [comment]
@toannguyen71195  : 
- start date should be before end date
- when clicking back while editing start date or end date, it saves the edited information instead of cancel

**Comment by toannguyen71195** [comment]
@vtnlinh95 @huuduy216 All fixed, please verify again

**Comment by hieumai** [comment]
@toannguyen71195  please rebase this so I can merge

**Comment by hieumai** [comment]
@toannguyen71195 solve conflicts

**Comment by hieumai** [comment]
@toannguyen71195 solve conflicts again

**Pull request merged**